cli/cmdcontext/cmdcontext.go

```diff
@@ -98,13 +100,13 @@ type RunningCtx struct {
 // ConnectCtx contains information for connecting to the instance.
 type ConnectCtx struct {
 	// Username of the tarantool user.
-	Username    string
+	Username string
```

Comment:
Not related with the commit. (And I really don't understand what you're fixing here)
Here and bellow.

Reply:
This is done automatically by go fmt.